src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -163,7 +163,8 @@ button.NormalAnnouncementBubble--deleteMenuItem {
 
 .NormalAnnouncementBubble--readReceipts--tooltip {
   .text--semi-bold();
-  padding-left: @size_s;
+  padding-right: @size_xs;
+  padding-left: @size_2xs;
```

Comment:
What was the thinking behind having more padding on the right than the left?

Reply:
I think the tooltip looked good for all readBy receipts except for when the array had 1 name. You can currently see how this looks on live, but the name seems too left-aligned so I added some padding on the right as well. There's a bit more padding on the right to make it seem more right-aligned. You can also observe this issue on live LP as well

Reply:
Okay sounds good, just want to double check that it looks better as is than if there was the same padding on both sides